src/V8.rs

```diff
@@ -61,24 +61,26 @@ where
   }
 }
 
-#[derive(Debug, Eq, PartialEq)]
+#[derive(Debug)]
 enum GlobalState {
   Uninitialized,
-  PlatformInitialized,
-  Initialized,
-  Disposed,
+  PlatformInitialized(SharedRef<Platform>),
+  Initialized(SharedRef<Platform>),
+  Disposed(SharedRef<Platform>),
   PlatformShutdown,
 }
 use GlobalState::*;
 
 lazy_static! {
-  static ref GLOBAL_STATE: Mutex<GlobalState> =
-    Mutex::new(GlobalState::Uninitialized);
+  static ref GLOBAL_STATE: Mutex<GlobalState> = Mutex::new(Uninitialized);
 }
 
 pub fn assert_initialized() {
   let global_state_guard = GLOBAL_STATE.lock().unwrap();
-  assert_eq!(*global_state_guard, Initialized);
+  match *global_state_guard {
+    Initialized(_) => {}
+    _ => panic!("Invalid state"),
```

Comment:
```suggestion
    _ => panic!("Invalid global state"),
```
For consistency with the other error messages.